admin/services/data-access/group.data.service.js

```diff
@@ -57,6 +57,28 @@ groupDataService.sqlFindOneById = async (groupId, schoolId) => {
   return R.head(result)
 }
 
+/**
+ * Get group by school id.
+ * @param schoolId
+ * @returns {Promise<void>}
+ */
+groupDataService.sqlFindOneBySchoolId = async (schoolId) => {
+  const sql = `SELECT * 
+    FROM ${sqlService.adminSchema}.[group]
+    WHERE school_id=@schoolId`
```

Comment:
@GeorgeChatzigiannis you may want to add isDeleted=0 to make sure you're not fetching deleted groups.